spec_2/scrabble.js

```diff
@@ -0,0 +1,57 @@
+const playedTiles = "AERETOXMYCNS_B"
```

Comment:
Sweet! Thanks so much for the feedback, @fryymann715!  Just getting into the flow of reviewing Prrrs; glad this was helpful.